Equations/src/main/java/db/helper/PlayerHelper.java

```diff
@@ -0,0 +1,33 @@
+package db.helper;
+
+import db.GameDAO;
+import db.UserDAO;
+import entity.GameRepresentation;
+import gamestatemanager.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlayerHelper {
+    private final GameDAO gameDAO;
+    private final UserDAO userDAO;
+
+    public PlayerHelper(GameDAO gameDAO, UserDAO userDAO) {
+        this.gameDAO = gameDAO;
+        this.userDAO = userDAO;
+    }
+
+    public List<Player> getPlayersByGameId(String gameId) {
```

Comment:
This is the method that does the database mapping for you. It is kinda messy but it works.